README.md

```diff
@@ -0,0 +1,129 @@
+    * `stats['content_crashes']` is the number of content process crashes represented by the aggregate.
+    * `stats['plugin_crashes']` is the number of plugin process crashes represented by the aggregate.
+    * `stats['gmplugin_crashes']` is the number of Gecko media plugin (often abbreviated GMPlugin) process crashes represented by the aggregate.
+
+Plugin process crashes per hour on Nightly for March 14:
+
+```sql
+SELECT sum(stats['plugin_crashes'] / sum(stats->>'usage_hours') FROM aggregates
+WHERE dimensions->>'channel' = 'nightly' AND submission_date = '2016-03-14'
+```
+
+Main process crashes by build date and E10S setting.
+
+```sql
+WITH channel_rates AS (
+  SELECT dimensions['build_date'] AS build_date,
+         SUM(stats['main_crashes']) AS main_crashes, -- total number of crashes
+         SUM(stats['usage_hours']) / 1000 AS usage_kilohours, -- thousand hours of usage
+         dimensions['e10s_enabled'] AS e10s_enabled -- e10s setting
+   FROM crash_aggregates
+   WHERE dimensions['experiment_id'] = 'None' -- not in an experiment
+     AND regexp_like(dimensions['build_date'], '^\d{14}$') -- validate build IDs
+     AND dimensions['build_date'] > '20160201000000' -- only in the date range that we care about
+   GROUP BY dimensions['build_date'], dimensions['e10s_enabled']
+)
+SELECT parse_datetime(build_date, 'yyyyMMddHHmmss') as build_date, -- program build date
+       usage_kilohours, -- thousands of usage hours
+       e10s_enabled, -- e10s setting
+       main_crashes / usage_kilohours AS main_crash_rate -- crash rate being defined as crashes per thousand usage hours
+FROM channel_rates
+WHERE usage_kilohours > 100 -- only aggregates that have statistically significant usage hours
+ORDER BY build_date ASC
+```
+
+Development
+-----------
+
+This project uses [Vagrant](https://www.vagrantup.com/) for setting up the development environment, and [Ansible](https://www.ansible.com/) for automation. You can install these pretty easily with `sudo apt-get install vagrant ansible` on Debian-derivative Linux distributions.
+
+To set up a development environment, simply run `vagrant up` and then `vagrant ssh` in the project root folder. This will set up a development environment inside a virtual machine and SSH into it. Do `cd /vagrant` to get to the project folder within the VM.
+
+To set up the environment locally on Ubuntu 14.04 LTS, simply run `sudo ansible-playbook -i ansible/inventory/localhost.ini ansible/dev.yml`.
+
+Note that within the Vagrant VM, you should use `~/miniconda2/bin/python` as the main Python binary. All the packages are installed for Miniconda's Python rather than the system Python.
+
+To backfill data, just run `crash_rate_aggregates/fill_database.py` with the desired start/end dates as the `--min-submission-date`/`--max-submission-date` arguments.
+
+To run the tests, execute `PYTHONPATH=.:$PYTHONPATH python -m unittest discover -s /vagrant/test` in the Vagrant VM.
```

Comment:
I think this might mess up other scripts in the future in subtle ways, if their imports suddenly go relative for no visible reason. Actually though, that variable def isn't even necessary, so I'll remove it.